### PR TITLE
Remove VisualStudio test suite infavour for nunit

### PR DESCRIPTION
**Pull request by Ragil**

### CODE REVIEW

File: src/Twilio.Lookups/Twilio.Lookups.csproj [hunk 1]
```diff
@@ -35,7 +35,7 @@
   <ItemGroup>
     <Reference Include="RestSharp, Version=105.2.2.0, Culture=neutral, processorArchitecture=MSIL">
```
Comment (jingming) [inline src/Twilio.Lookups/Twilio.Lookups.csproj]: should these two match?

File: src/Twilio.Monitor/Twilio.Monitor.csproj [hunk 2]
```diff
@@ -33,7 +33,7 @@
   <ItemGroup>
     <Reference Include="RestSharp, Version=105.2.2.0, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\RestSharp.105.2.2\lib\net35\RestSharp.dll</HintPath>
+      <HintPath>..\packages\RestSharp.105.2.3\lib\net35\RestSharp.dll</HintPath>
```
Comment (jingming) [inline src/Twilio.Monitor/Twilio.Monitor.csproj]: same here

Comment (Ragil) [inline src/Twilio.Monitor/Twilio.Monitor.csproj]: good catch

File: src/Twilio.Lookups/Twilio.Lookups.csproj (hunk 1 above)
Comment (Ragil) [inline src/Twilio.Lookups/Twilio.Lookups.csproj]: good catch

**Comment by jingming** [comment]
:+1: :+1:

**Pull request merged**